mage2-fpm/Dockerfile

```diff
@@ -8,6 +8,8 @@ RUN mkdir -p /usr/local/bin \
     && curl -s https://files.magerun.net/n98-magerun2.phar > /usr/local/bin/n98-magerun \
     && chmod +x /usr/local/bin/n98-magerun
 
+RUN curl -o /etc/bash_completion.d/n98-magerun2.phar.bash https://raw.githubusercontent.com/netz98/n98-magerun2/master/res/autocompletion/bash/n98-magerun2.phar.bash
```

Comment:
```suggestion
RUN curl -o /etc/bash_completion.d/n98-magerun2.phar.bash \
        https://raw.githubusercontent.com/netz98/n98-magerun2/master/res/autocompletion/bash/n98-magerun2.phar.bash \
    && perl -pi -e 's/^(complete -o default .*)$/$1 n98-magerun/' /etc/bash_completion.d/n98-magerun2.phar.bash
```

Reply:
Added perl command here to add `n98-magerun` to list of completions served by the command since this image has `n98-magerun` and not `n98-magerun2`